client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -191,6 +201,11 @@ public void testVarbinary()
                         .hasMessageStartingWith("Value is not a number: [B@");
 
                 assertEquals(rs.getString(column), "0x12345678");
+                assertThat(rs.getBinaryStream(column)).hasBinaryContent(bytes);
```

Comment:
add `rs.getAsciiStream` check here too; it can throw
however i don't think it should return stream of `0x12345678` characters (as rs.getString does).

Reply:
added limit to getAsciiStream